Gifski/VideoValidator.swift

```diff
@@ -184,6 +184,20 @@ struct VideoValidator {
 			return .failure
 		}
 
-		return .success(newAsset, newVideoMetadata)
+		// Trim asset
+		do {
+			let trimmedAsset = try newAsset.trimmingBlankFrames()
+
+			return .success(trimmedAsset, newVideoMetadata)
+		} catch {
+			NSAlert.showModalAndReportToCrashlytics(
+				for: window,
+				title: "Could not trim empty leading frames from video.",
+				message: "Error: \(error.localizedDescription). This should not happen. Email sindresorhus@gmail.com and include this info:",
```

Comment:
```suggestion
				message: "\(error.localizedDescription)\n\nThis should not happen. Email sindresorhus@gmail.com and include this info:",
```

Reply:
Example of how it looks now:
![Screen Shot 2021-06-24 at 2 01 43 PM](https://user-images.githubusercontent.com/7410405/123318718-23585680-d4f5-11eb-9009-1559d84562ab.png)